### DISCUSSION TITLE
Question regarding multiple scanners

### DISCUSSION THREAD

**Discussion by rscafidi** [Forceu/barcodebuddy#239] [Q&A]
I searched the issues but didn't see this, apologies if it's already been addressed.  Does barcode buddy support multiple scanners if we are running the docker version?  Or can you only use one?  I would like to have a few scanners in different locations, all connected to the barcode buddy instance I have running but curios if it would support that before I buy a second scanner.

**Comment by bgriffen** [comment]
Seconded - also interested

**Comment by Forceu** [comment]
Hi, sorry for the late reply.
The docker image itself only supports one barcode scanner, however you can run the script that attaches to the devices locally and then post the barcodes to the docker instance. Use either [this](https://github.com/Forceu/barcodebuddy/blob/master/example/grabInput.sh) or [this](https://github.com/Forceu/barcodebuddy/blob/master/example/grabInput.py) script and change the variable for the destination and insert your API key
